README.rst

```diff
@@ -1,6 +1,7 @@
-.. image:: https://docs.pytest.org/en/stable/_static/pytest1.png
+.. image:: https://github.com/nicoddemus/pytest/raw/transparent-logo-8158/doc/en/img/pytest_logo_curves.svg
```

Comment:
This needs to be updated before merging.

Reply:
Also: you may want to update the `conf.py` to use SVG and see how it goes.